js/ui/toolbar/ui.toolbar.base.js

```diff
@@ -149,12 +149,38 @@ var ToolbarBase = AsyncCollectionWidget.inherit({
         this.setAria("role", "toolbar");
     },
 
+    _waitParentAnimationFinished: function() {
+        var $element = this.$element(),
+            timeout = 15;
+        return new Promise(function(resolve) {
+            const check = function() {
+                var animated = true;
+                $element.parents().each(function(_, parent) {
+                    if($(parent).data("dxAnimData")) {
+                        animated = false;
+                    }
+                });
+                animated && resolve();
+                return animated;
+            };
+            const runCheck = function() {
+                setTimeout(function() {
+                    check() || runCheck();
+                }, timeout);
+            };
+            ($element.width() > 0 && check()) || runCheck();
+        });
+    },
+
     _render: function() {
+        var that = this;
         this.callBase();
         this._renderItemsAsync();
 
         if(themes.isMaterial()) {
-            this._checkWebFontForLabelsLoaded().then(this._dimensionChanged.bind(this));
+            this._waitParentAnimationFinished().then(function() {
```

Comment:
We can use an arrow function. We do not need 'that' in this case. My suggestion is `Promise.all([this._waitParentAnimationFinished(),this._checkWebFontForLabelsLoaded()]).then(this._dimensionChanged.bind(this))`